test/SlowTests/Issues/RavenDB_12601.cs

```diff
@@ -160,10 +167,12 @@ public async Task Can_use_cluster_tx_mixed_with_tx()
 
                     session.Store(user);
                     session.SaveChanges();
-
+                    
+                    trxn += GetNumOfUpdateLicenseLimitsCommand(Server); // could be 1 or more 'UpdateLicenseLimitsCommand' before the 'ClusterTransactionCommand'
```

Comment:
So this is still a racy.
Better to wait for the expected number of `UpdateLicenseLimitsCommand`.

Reply:
or for the expected raft index

Reply:
Done